examples/nextjs-with-typescript/src/StylesContext.tsx

```diff
@@ -0,0 +1,9 @@
+import { createContext } from 'react';
+
+export interface StyleContextData {
+  key: string;
+  ids: Array<string>;
+  css: string;
+}
+
+export default createContext<null | StyleContextData[]>(null);
```

Comment:
Is this the correct place? it is in `nextjs-with-typescript`

Reply:
Ooopsy, I should remove it. Must have copied it in the wrong folder. Good catch!